src/share/vm/jfr/jni/jfrJavaSupport.cpp

```diff
@@ -56,6 +57,12 @@ void JfrJavaSupport::check_java_thread_in_native(Thread* t) {
   assert(t->is_Java_thread(), "invariant");
   assert(((JavaThread*)t)->thread_state() == _thread_in_native, "invariant");
 }
+
+static void check_new_unstarted_java_thread(Thread* t) {
+  assert(t != NULL, "invariant");
+  assert(t->is_Java_thread(), "invariant");
+  // assert(((JavaThread*)t)->thread_state() == _thread_new, "invariant");
```

Comment:
This line is in jdk master. Why comment this line? please add a comment here?

Reply:
fixed